gcloud/test_credentials.py

```diff
@@ -46,6 +46,7 @@ def _setup_appengine_import(test_case, app_identity):
     sys.modules['google.appengine.ext'] = GAE_EXT
     sys.modules['google.appengine.ext.webapp'] = GAE_EXT_WEBAPP
     sys.modules['google.appengine.ext.webapp.util'] = GAE_EXT_WEBAPP_UTIL
+    sys.modules['webapp2'] = GAE_EXT_WEBAPP
```

Comment:
More encapsulation leakage:  auditing more deeply exactly how `oauth2client` is doing its imports, it looks like it still expects `google.appengine.ext.webapp.util` to be importable (which begs the question, "Why can we not rely then on `google.appengine.ext.webapp`?).